src/routes/(app)/+page.svelte

```diff
@@ -20,3 +21,5 @@
         >the Wiki</a
     > to read the documentation
 </p>
+
+<Post />
```

Comment:
I know we're wanting to display that squirrel video, so I'd rather we hardcode that here than within the component

Reply:
Also in latest commit.